.test-conda-env-py3.yml

```diff
@@ -2,6 +2,7 @@ name: test-conda-env-py3
 channels:
 - conda-forge
 - nodefaults
+- cantera/label/dev
```

Comment:
This should no longer be necessary. Cantera is in conda forge proper.

Reply:
This is something I should address in #236